src/backend/integration/src/support/wasm.ts

```diff
@@ -18,7 +18,10 @@ export const BACKEND_WASM_PATH = resolve(
 
 export async function setupBackendCanister(
   pic: PocketIc,
+  initialDate: Date = new Date(),
 ): Promise<CanisterFixture<_SERVICE>> {
+  await pic.setTime(initialDate.getTime());
```

Comment:
We need to set the date explicitly before deploying the canister because the default admin profile is added on `init` and we want to have a predictable date for that in the profile history.